packages/slate-react/src/plugins/after.js

```diff
@@ -267,15 +267,17 @@ function AfterPlugin() {
     const { document } = value
     const node = findNode(event.target, value)
     const isVoid = node && (node.isVoid || document.hasVoidParent(node.key))
+    const selectionIncludesNode = value.blocks.some(
+      block => block.key === node.key
+    )
 
-    if (isVoid) {
-      const encoded = Base64.serializeNode(node, { preserveKeys: true })
-      setEventTransfer(event, 'node', encoded)
-    } else {
-      const { fragment } = value
-      const encoded = Base64.serializeNode(fragment)
-      setEventTransfer(event, 'fragment', encoded)
+    if (isVoid && !selectionIncludesNode) {
```

Comment:
Can you add a comment to explain what's going on here, why this case would ever come up?